### ISSUE TITLE
Do not automatically install Artifactory backend

### ISSUE THREAD

**Issue by frankenjoe** [audeering/audbackend#135]
Since it requires `dohq-artifactory`, I wonder if we should not provide `Artifactory` by default but only if explicitly installed with `install audbackend[artifactory]`.

**Comment by frankenjoe** [comment]
Since we anyway introduce some breaking changes, it's a good moment to do that.

**Comment by hagenw** [comment]
Yes, sounds very reasonable to me as you might never want to use it.

In packages for which we think it will be to complicated if a user has to ensure that the Artifactory backend is present (`audb`) we could simply continue to install it as default.
